specs/language/expressions.tex

```diff
@@ -204,9 +205,11 @@
 undefined behavior to not explicitly initialize an \textit{output parameter}
 before returning from the function in which it is defined. The cxvalue created
 from an argument to an \textit{input/output parameter} is initialized through
-copy-initialization from the lvalue argument expression. In both cases, the
-cxvalue shall have the type of the parameter and the argument can be converted
-to that type through implicit or explicit conversion.
+copy-initialization from the lvalue argument expression, overload resolution may
+occur on argument initialization as if the expression \texttt{T Param = Arg}
+were evaluated. In both cases, the cxvalue shall have the type of the parameter
```

Comment:
This is a bit of a malformed sentence. Either it's a run-on or missing the word "if" before "The cxvalue created".

Also small nit: I think it should be "was evaluated", not "were evaluated", as "expression" is singular.

Reply:
I wonder if a `;` instead of a `comma` on line 208 might also fix it?  Or maybe it should just be two sentences.